source/javascripts/explorer/asana-explorer.js

```diff
@@ -667,7 +667,7 @@ var Explorer = /** @class */ (function (_super) {
 })(Explorer || (Explorer = {}));
```

Comment:
Do we still need this file if it's not being used? Was this file auto generated?

Reply:
it's being used in `popup_receiver.html`: https://github.com/Asana/developer-docs/blob/master/source/explorer/popup_receiver.html#L5

i'll switch it to use the min version too and then i think we can just get rid of this file

Reply:
I guess it's still being referenced in `popup_receive.html` https://github.com/Asana/developer-docs/blob/master/source/explorer/popup_receiver.html#L5